src/get-started/java-binding.md

```diff
@@ -472,6 +477,72 @@ client.
 An example of API service tests can be found in
 [`ApiControllerTest`][apicontrollertest].
 
+<!-- TODO: Where it belongs? Do we need such instructions in these docs, or just have
+them in the generated project POM? -->
+## Using Libraries
+
+An Exonum service can use any third-party library as its dependency.
+
+<!-- 
+Todo: Which of the following? The first, though complete, is a little wordy.
+
+The libraries, classes from which are used in Exonum public APIs, 
+are provided with the framework distribution, and can also be used by the service.
+
+---
+
+Some libraries are provided with the framework distribution, and can also be used
+by the service.
+-->
+
+In this case, they *must not* be packaged in its artifact. Use `provided` scope
+in the dependency declaration to achieve that in Maven.
+
+The following dependencies **must** be used as provided, because types from these
+libraries appear in Exonum public APIs. They will not be changed in an incompatible
+way in a compatible Exonum release:
+- Exonum <!-- TODO: specify exact artifacts? -->
```

Comment:
I would replace strings by links to maven central or projects home pages

Reply:
Projects home pages 👍